CHANGELOG.rst

```diff
@@ -2,10 +2,19 @@
 Changelog
 =========
 
-1.2.0 (2022-??-??) - in development
+2.0.0 (2022-??-??) - in development
 ===================================
 
-BEFORE upgrading to 1.2.x, please read the release notes carefully.
+BEFORE upgrading to 2.0.0, please read the release notes carefully.
+
+**Upgrade procedure**
```

Comment:
This section will be expanded in the issue https://github.com/open-formulieren/open-forms/issues/1922 ?

Reply:
#1895 rather, but yeah, this is a placeholder for now :-)